app/components/pages/Login/index.js

```diff
@@ -3,14 +3,13 @@ import { Flex, Box } from 'grid-styled'
 import { Redirect } from 'react-router-dom'
 import { withApollo } from 'react-apollo'
 import gql from 'graphql-tag'
-import config from 'config'
 import { H2 } from '@pubsweet/ui'
 import OrcidButton from '../../ui/atoms/OrcidButton'
 import ButtonLink from '../../ui/atoms/ButtonLink'
 import Paragraph from '../../ui/atoms/Paragraph'
 import ExternalLink from '../../ui/atoms/ExternalLink'
 
-const { url: loginUrl, signupUrl, legacySubmissionUrl } = config.login
+const { url: loginUrl, signupUrl, legacySubmissionUrl } = window.config.login
```

Comment:
How about using [Webpack externals](https://webpack.js.org/configuration/externals/) to avoid this change? If not, you'll need to update the other places where `config` is used in the front-end (and it would break theoretical compatibility with PubSweet components).

Reply:
Should we use Webpack externals to replace `common-resolve.js`?

Reply:
Delete! Delete!